assets/src/conversion/index.js

```diff
@@ -58,22 +70,50 @@ class Conversion extends Component {
 		} );
 	};
 
+	handleOnClickInitializeRetryFailed = () => {
+		return fetchInitializeRetryFailedConversion().then( response => {
+			if ( ! response || ! response.result || 'queued' != response.result ) {
+				return new Promise( ( resolve, reject ) => resolve() );
```

Comment:
1. I don't understand why a Promise is returned here (and two lines above), since it's just a click handler 🤔(same in `handleOnClickInitializeConversion` above)
2. tip: a Promise that resolves immediately can be just [`Promise.resolve()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve)

Reply:
True, returning Promises isn't necessary in these two cases. Updated to return a simple void.